### DISCUSSION TITLE
[Feature] iOS app - Disable backups completely

### DISCUSSION THREAD

**Discussion by sebastian-ruiz** [immich-app/immich#3296] [Feature Request]
### The feature

I use Nextcloud to backup my photos and I am using the CLI to give Immich read access. 

It would be nice if there was a clearer toggle switch to disable backups completely.

![IMG_2087](https://github.com/immich-app/immich/assets/2012398/0283c4e2-f4d9-42c5-b8cc-8d6969d0294f)

I have 
"Automatic foreground backup is off"
and
"Automatic background backup is off"

but from the UI it still looks like it wants to backup my photos.

Thanks! I love the app :)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Peckmore** [comment]
I'd really like this feature too, but I'd love to be able to control it server-side rather than client-side.

I have family who I'd love to give the mobile app to so they could view albums I share with them, but neither of us want to backup their photos to my server. It would be great if I could turn off backup support server-side on a per user basis so then their app would hide any functionality related to backing up photos, and effectively just become a viewer for shared albums.

I think (at least for the mobile app) this would mostly be a case of hiding the cloud/upload button at the top-right of the Timeline view, which I think would hide all upload related functionality?

**Comment by Todd1561** [comment]
Same request, for the same reason.  I have my own solution for getting photos off my devices, in fact I only use Immich with a single external library using the recent feature addition (fantastic by the way!)  For now, my workaround to ensure Immich doesn't do any mass uploading is to use the iOS feature to limit the Immich app access to a single photo.  This has the unfortunate side-effect of displaying a popup in the app upon launching asking for more access to photos, to which I decline.

You guys have done amazing work here, this app does everything Google Photos ever did for me and I get to host it myself.  Thanks for your hard efforts, I'll definitely be making a donation to help keep this going.

**Comment by driesmp** [comment]
Yes please! A disable upload toggle option or something :) ideally server side!

**Comment by Joewawaw** [comment]
Hey, bumping this back up, I agree with @Peckmore, would it be possible to add an option for each user, on the server side, to disable uploads? (i.e. backups on the android/ios app and uploads on the web ui). This would allow immich to become a "photo viewer" app for those we do not want to allow uploading.

**Comment by franchyze923** [comment]
I would also like this feature!

**Comment by bo0tzz** [comment]
This now exists

**Reply by davru** [reply]
where is this feature? I want to avoid a user doing auto backups from any device he connects from but cannot find the way to toggle it

**Reply by bo0tzz** [reply]
It's just the enable backups toggle in the app's backup screen. That's not the same thing as what you're looking for. You should probably just set the user's quota very low.
